### PR TITLE
Added Database Schema

### PR DESCRIPTION
**Pull request by DeboDevelop**
## What?

Added the following database schema:
1. ChatBot.
2. Intent.
3. Utterance.
4. Response.

## Testing?

Done in the local environment.

## Screenshots/Recording (optional)

Chatbot:
Screenshot from 2022-11-12 13-49-38

Intent:
Screenshot from 2022-11-12 13-51-45

Utterance:
Screenshot from 2022-11-12 13-52-37

Response:
Screenshot from 2022-11-12 13-53-55

### CODE REVIEW

**Review by AdityaTiwari2102 — APPROVED** [review]

File: app/models/intent.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class Intent < ApplicationRecord
```
Comment (AdityaTiwari2102) [inline app/models/intent.rb:1]: @DeboDevelop - Associate the intent with a chatbot as well.

File: app/models/chat_bot.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+class ChatBot < ApplicationRecord
```
Comment (AdityaTiwari2102) [inline app/models/chat_bot.rb]: @DeboDevelop - Chatbot is a single word.

File: db/migrate/20221111141747_create_utterances.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+class CreateUtterances < ActiveRecord::Migration[7.0]
+  def change
+    create_table :utterances do |t|
+      t.text :utterance_text
```
Comment (AdityaTiwari2102) [inline db/migrate/20221111141747_create_utterances.rb]: @DeboDevelop - Can we rename it to `content`?

**Review by AdityaTiwari2102 — CHANGES_REQUESTED** [review]

File: app/models/intent.rb (hunk 1 above)
Comment (DeboDevelop) [reply app/models/intent.rb:1]: @AdityaTiwari2102 Done.

File: app/models/chat_bot.rb (hunk 2 above)
Comment (DeboDevelop) [reply app/models/chat_bot.rb]: @AdityaTiwari2102 Fixed.

File: db/migrate/20221111141747_create_utterances.rb (hunk 3 above)
Comment (DeboDevelop) [reply db/migrate/20221111141747_create_utterances.rb]: @AdityaTiwari2102 Done.

**Review by AdityaTiwari2102 — APPROVED** [review]

**Pull request merged**